eessi/testsuite/tests/apps/lammps/lammps.py

```diff
@@ -8,9 +8,39 @@
 import reframe.utility.sanity as sn
 
 from eessi.testsuite import utils
-from eessi.testsuite.constants import COMPUTE_UNITS, DEVICE_TYPES, TAGS
+from eessi.testsuite.constants import COMPUTE_UNITS, DEVICE_TYPES, TAGS, SCALES
 from eessi.testsuite.eessi_mixin import EESSI_Mixin
 
+from statistics import mean
+
+# Todo should find a way to set the tag CI when the module of LAMMPS is not a fat-build
+# The only way to easily check it without running lmp is to check the easyconfig in software dir
+
+
+def split(list, size):
+    return [list[i:i + size] for i in range(0, len(list), size)]
+
+def filter_scale_cores():
+    """
+    Filtering function for filtering scales for the LAMMPS ALL small test
+    returns all scales with 8 or less cores.
+    """
+    print(SCALES.items())
+    return [
+        k for (k, v) in SCALES.items()
+        if (k in ['1_core', '2_cores', '4_cores', '8_cores', '1cpn_2nodes', '1cpn_4nodes'])
```

Comment:
I think we should do this in a different way that is more robust against `SCALES` potentially being added later. 
```suggestion
        if ('num_cpus_per_node' in v.keys() and v['num_cpus_per_node'] * v['num_nodes'] <= 8)
```
(e.g. this would still work if we ever decided to add a `1cpn_8nodes` scale for example)